src/views/my-account/my-account.view.jsx

```diff
@@ -62,6 +73,13 @@ function MyAccount ({
 
   return (
     <div className={classes.root}>
+      {
+        estimatedRewardTask.status === 'successful' && earnedRewardTask.status === 'successful' 
+          ? <Sidenav onClose={onClose}>
```

Comment:
I would render the `Sidenav` component inside the `AirdropPanel` component.

Reply:
Does not make sense to me :) Can you explain why?

Reply:
Because if you need to render the `Sidenav` every time you render the `AirdropPanel` view is a good sign of include it inside the component. So you can avoid render the `Sidenav` component apart every time you want to use the `AirdropPanel` view and handle its callbacks etc

Reply:
Ah I understand. I misread it... sorry.

Reply:
Will change.

Reply:
No prob :) Nice!